packages/next/src/export/worker.ts

```diff
@@ -251,6 +251,7 @@ async function exportPageImpl(
         page,
         components.routeModule as AppRouteRouteModule,
         input.renderOpts.incrementalCache,
+        input.renderOpts.cacheLifeProfiles,
```

Comment:
Can Infinity be serialized across worker bounds or do we need to handle this custom?

Reply:
As long as we use the built-in structured clone default, then yes. As long as we don't turn it into JSON manually.